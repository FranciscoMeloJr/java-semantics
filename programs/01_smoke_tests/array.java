public class main {

	public static void main(String[] args) {
		int[] v = new int[1];
		v[0] = 1;
		v[0]++;
		System.out.println("v[0]="+ v[0]);
    System.out.println("Done!");
	}
}

// v[0]=2
// Done!
