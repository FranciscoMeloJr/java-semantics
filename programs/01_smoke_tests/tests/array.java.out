v[0]=2
Done!
